driver-core/src/main/com/mongodb/AutoEncryptionSettings.java

```diff
@@ -26,15 +26,39 @@
 import static com.mongodb.assertions.Assertions.notNull;
 
 /**
- * The auto-encryption options.
+ * The client-side automatic encryption settings. Client side encryption enables an application to specify what fields in a collection
```

Comment:
This class is supposed to be prefixed with ClientSideEncryption, but then what do we call the setting class for the ClientEncryption class itself.  I see in the current spec version we did an end run around that by using constructor parameters, but that's inconsistent with the MongoClient pattern.  

CC @kevinAlbs

Reply:
Yeah, ClientEncryption should take a ClientEncryptionSettings,  and MongoClient should  take AutoEncryptionSettings, like you have now. I need to update the spec.